e2e/testconfig/testconfig.go

```diff
@@ -132,5 +138,38 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string) ibc.Cha
 		GasAdjustment:  1.3,
 		TrustingPeriod: "508h",
 		NoHostMount:    false,
+		ModifyGenesis:  modifyGenesisVotingPeriod("10s", denom),
+	}
+}
+
+func modifyGenesisVotingPeriod(votingPeriod string, denom string) func([]byte) ([]byte, error) {
```

Comment:
I will split this out and clean it up once I figure out how to unmarshal the app state without using dyno